app/src/lib/dispatcher/app-store.ts

```diff
@@ -1181,17 +1179,6 @@ export class AppStore {
   }
 
   /** This shouldn't be called directly. See `Dispatcher`. */
-  public async _validatedRepositoryPath(path: string): Promise<string | null> {
```

Comment:
Lol yeah that was kinda my reaction. I assume it's from Earlier Days when we had less of an idea what we were doing.